certbot-ci/certbot_integration_tests/nginx_tests/context.py

```diff
@@ -29,15 +29,16 @@ def cleanup(self):
         self._stop_nginx()
         super(IntegrationTestsContext, self).cleanup()
 
-    def certbot_test_nginx(self, args):
+    def certbot_test_nginx(self, args, force_renew=True):
```

Comment:
Same comment about `force_renew` being set this way as above.

Reply:
See above

Reply:
In fact not, not as above. Indeed I am passing positionally the optional argument on nginx tests. Bad me.

Checking more closely, in fact I think we can always renew by default, because each second call to Certbot is about rolling back the nginx config, not creating a new certificate. But never renew by default will also work, since each first call to Certbot is for a new domain. 

What about just removing the parameter in `certbot_test_nginx`, and always renew by default (this instead of no renew by default to be consistent with the major case in certbot tests).

Also, why not calling directly `certbot_test` in this function instead of `_common_test` from parent class? Then we could remove `_common_test` completely.

Reply:
I did the two propositions (always force renew in nginx, remove `_common_test`.